masterlist.yaml

```diff
@@ -775,7 +867,10 @@ plugins:
     tag:
       - Delev
   - name: 'aMidianBorn_ContentAddon.esp'
-    url: ['https://www.nexusmods.com/skyrim/mods/24909/']
+  # aMidianBorn Book of Silence
```

Comment:
No need for the comment since you've added a named link.

Reply:
Removed it